### PR TITLE
Upgrade to 3.7

### PR DESCRIPTION
**Pull request by wolfie90**
You might have to consider also updating it on the Fork CMS website.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/.idea
```
Comment (WouterSioen) [inline .gitignore]: This should be in your global .gitignore file, not in the project. (https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

File: VERSION.md [hunk 2]
```diff
@@ -0,0 +1 @@
+2.0.0
```
Comment (WouterSioen) [inline VERSION.md:1]: In the changelog, you use 2.2.0 and in this file 2.0.0?

**Comment by WouterSioen** [comment]
Thanks for the update. Some comments:

It looks like you used updated the file structure and not the coding styles?

Also, a little strange to pull request a licence containing your credentials for a repo you don't manage?

File: src/Backend/Modules/Twitter/Actions/Edit.php [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+					// create
+					$userId = BackendTwitterModel::insertUser(array('username' => $txtUsername->getValue()));
+
+					// set oauth flag
+					$mustAuthenticate = true;
+				}
+
+				// item data
+				$item['id'] = $this->id;
+                $item['extra_id'] = $this->record['extra_id'];
```
Comment (WouterSioen) [inline src/Backend/Modules/Twitter/Actions/Edit.php]: inconsistent indentation

**Comment by wolfie90** [comment]
Ah yeah I think I'll have to take a deeper look at the coding styles. I think I accidentally also copied the LICENSE.md from my other modules, will remove that :+1: .

**Comment by WouterSioen** [comment]
Try php-cs-fixer, it's a great tool to make it easier to convert to psr.
Something like `php-cs-fixer fix --level=psr2 .` should fix most of the issues.

**Comment by wolfie90** [comment]
Fixed the issues you mentioned :smile:

**Comment by WouterSioen** [comment]
Thanks, now it'll be easier to do a more in depth code review :)

File: src/Backend/Modules/Twitter/Actions/Add.php [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+use Backend\Core\Engine\Base\ActionAdd as BackendBaseActionAdd;
+use Backend\Core\Engine\Form as BackendForm;
+use Backend\Core\Engine\Language as BL;
+use Backend\Core\Engine\Model as BackendModel;
+use Backend\Modules\Twitter\Engine\Model as BackendTwitterModel;
+
+/**
+ * This is the add-action, it will display a form to create a new item
+ *
+ * @author Bert Pattyn <bert@netlash.com>
```
Comment (WouterSioen) [inline src/Backend/Modules/Twitter/Actions/Add.php:21]: Where did you get this module from?

**Comment by timvanwolfswinkel** [comment]
From the Fork CMS website, why? Is it missing something? Thought it would be the same version.

File: src/Backend/Modules/Twitter/Actions/Add.php (hunk 4 above)
Comment (wolfie90) [inline src/Backend/Modules/Twitter/Actions/Add.php:21]: From the Fork CMS website, why? Is it missing something? Thought it would be the same version.

**Comment by WouterSioen** [comment]
No, it just looks like you're not updating the module But replacing it by a
completely different one.

Op donderdag 8 januari 2015 heeft Tim van Wolfswinkel <
notifications@github.com> het volgende geschreven:

**Comment by wolfie90** [comment]
Could it be that this is happening because the structure and names of files (capitals) have been changed? We succesfully used the old module in several projects. I also tested this upgraded module and it seems to be working ok again :). Let me know if something comes up!

**Comment by WouterSioen** [comment]
I think you've updated this Twitter module: https://github.com/forkcms/module-twitter and not the SumoCoders version.

**Comment by wolfie90** [comment]
I think you are right, did not find this one on Github :flushed:. What are the differences between these two? I think you could remove this Pull Request then and I will make a different one for that repository?

**Comment by WouterSioen** [comment]
The Sumocoders version didn't write any data to the database. It just fetched the tweets every 15 minutes and cached them using SpoonTemplate cache. The ForkCMS version saves everything in the database.

**Comment by wolfie90** [comment]
Ah ok! I'm sorry for the inconvenience and confusion..

**Comment by WouterSioen** [comment]
No problem, it's good to see someone updating modules :wink:

**Comment by wolfie90** [comment]
Oh yeah and thanks for the tips on the code :+1: !

**Pull request closed without merging**